backend/frontend/templates/frontend/user/board/page_card_details.html

```diff
@@ -0,0 +1,117 @@
+{% extends "frontend/user/base.html" %}
```

Comment:
We don't want a page called card details. 

It is not a part of the board, it should not be inside the board directory. It should also not be inside the user directory.

If we totally delete the card then the project will still exist. We need to access projects that have no card associated with them.  Same for topics. So this is not a card.

We also don't always access these pages via the board. We can get to them from a code review dashboard or similar.

A better path would be something like:

`backend/frontend/templates/frontend/{something}`

Either we would need separate pages for topic and projects in which case we would have:

`backend/frontend/templates/frontend/page_project_details`

Or we would need to combine both of the topic and project pages into one template in which case this might make more sense:

`backend/frontend/templates/frontend/page_course_component_details` or something like that.